pkg/stdlib/sql.go

```diff
@@ -208,18 +224,6 @@ func (c *Conn) QueryContext(ctx context.Context, query string, argsV []driver.Na
 	return &Rows{conn: c, rows: queryResult.Rows}, nil
 }
 
-func (c *Conn) Ping(ctx context.Context) error {
```

Comment:
Why this was removed?

Reply:
Implementing pinger interface Is not needed in the initial implementation.